.github/workflows/sync_branches.yml

```diff
@@ -0,0 +1,28 @@
+name: Sync Branches
+on:
+  push:
+    - "2.4"
+    - "2.5"
+
+jobs:
+  sync-branches:
+    strategy:
+      matrix:
+        include:
+          - from_branch: 2.4
+            to_branch: 2.5
+          - from_branch: 2.5
+            to_branch: main
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_name == matrix.to_branch }}
+    steps:
+      - name: Merge ${{ matrix.from_branch }} into ${{ matrix.to_branch }}
+        uses: tretuna/sync-branches@1.4.0
```

Comment:
Looking at the low popularity of that repo, you might want to pin the commit instead of a lose tag to prevent potential security risks